docs/building-your-quest/triggers-and-payloads.md

```diff
@@ -118,6 +118,21 @@ trigger:
         - actionId: a different action here
 ```
 
+### USER_EVENT
+
+Triggered when there's a custom event called on the user, you can trigger it by calling `https://engine.wilco.gg/users/${WILCO_USER_ID}/event`. You can find the user id in the `.wilco` file in the user's repository.
```

Comment:
I think it can be useful to add an example of an event payload we expect to receive